lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -102,9 +103,47 @@
           "type": "array",
           "items": {
             "type": "cid-link",
-            "description": "List of new blobs (by CID) referenced by records in this commit."
+            "description": "DEPRECATED -- will soon always be empty. List of new blobs (by CID) referenced by records in this commit."
           }
         },
+        "prevData": {
+          "type": "cid-link",
+          "description": "The root CID of the MST tree for the previous commit from this repo (indicated by the 'since' revision field in this message). Corresponds to the 'data' field in the repo commit object. NOTE: this field is effectively required for the 'inductive' version of firehose."
+        },
+        "time": {
+          "type": "string",
+          "format": "datetime",
+          "description": "Timestamp of when this message was originally broadcast."
+        }
+      }
+    },
+    "sync": {
+      "type": "object",
+      "description": "Updates the repo to a new state, without necessarily including that state on the firehose. Used to recover from broken commit streams, data loss incidents, or in situations where upstream host does not know recent state of the repository.",
+      "required": ["seq", "did", "blocks", "rev", "data", "time"],
+      "properties": {
+        "seq": {
+          "type": "integer",
+          "description": "The stream sequence number of this message."
+        },
+        "did": {
+          "type": "string",
+          "format": "did",
+          "description": "The account this repo event corresponds to. Must match that in the commit object."
+        },
+        "blocks": {
+          "type": "bytes",
+          "description": "CAR file containing the commit, as a block. The CAR header must include the commit block CID as the first 'root'.",
+          "maxLength": 10000
+        },
+        "rev": {
+          "type": "string",
+          "description": "The rev of the commit. This value must match that in the commit object."
+        },
+        "data": {
```

Comment:
flagging for discussion: should we actually break this out top-level, or leave it in the commit? `rev` is the important one to have top-level.